spec/requests/post_request_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/:user_id/posts' do
+    it 'Show a list of posts' do
+      user = User.create(name: 'Example')
+      get user_posts_path(user)
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'GET /users/:user_id/posts/:id' do
+    it 'show a single post' do
+      user = User.create(name: 'Example')
+      post = user.posts.create(title: 'Title Example')
+      get user_post_path(user, post)
+      expect(response).to have_http_status(200)
+    end
+  end
```

Comment:
Nice job adding a test for the response status😉. but just as you have implemented in the `user_request_spec.rb` kindly add a test to;
- show the right template is rendered.
- show the response body that includes correct placeholder text.
don't forget, this is a requirement in this milestone👍🏼.

<img width="903" alt="image" src="https://github.com/Zven94/BlogApp/assets/64643995/3cd000a8-e6c4-4e6e-a725-f62877b2d097">